node/nodejs_intro/README.md

```diff
@@ -69,55 +102,78 @@ Let's take a look at the official Chrome V8 website: [https://developers.google.
+<details>
+<summary>
+    Important Note
+  </summary>
+  😊 You're doing great! 😊
+</details>
+<p>&nbsp;</p>
 
+# Objective 3: The Event Loop
 
-# 4. The Event Loop
+You might have heard that JavaScript is `single-threaded` , this means it can only do one task at a time, and must wait for that task to finish before picking up another one. You may have also heard that it's [`asynchronous`](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Asynchronous/Concepts), meaning while it waits on a longer task to finish, it can pick up a different task. Well, hold on, that doesn't make sense, how could it be both?! Good question!
 
-You've probably heard that JavaScript is single-threaded, but you've also heard that it's asynchronous. This is probably when you start to say, "that doesn't make sense, how could it be both?". Good question! Answer: JS is single-threaded, the asynchronous behavior is not part of the JS language itself, but is actually built on top of the core JS language in the browser (or other programming environment) and is accessed through the browser APIs.
+Answer: JS itself is 🧵 `single-threaded` 🧵, and it's asynchronous behavior is built on top of the core JS language in the browser (or other programming environment) and is accessed through the browser APIs. [EDIT]
 Take a look at the following picture:
 
-![eventLoopImage](https://i.imgur.com/rnQEY7o.png)
+![eventLoopImage](assets/eventloop.png)
+*Event Loop via [link](https://i.imgur.com/rnQEY7o.png)*
 
-I know it looks pretty weird, but let's break down the different parts of the picture and see if we can get it to make more sense. For now, disregard the heap part.
+Let's break down the different parts of the picture and see if we can get it to make more sense. For now, disregard the heap part.
 
-The outer box is the Google Chrome that we know and love. For this example that is our programming environment.
+- The outer box is the Google Chrome that we know and love. For this example that is our programming environment.
+![Event Loop Chrome](assets/eventloopchrome.png)
 
-The next box we see is the JS box. This is where our code runs. The call stack is where we are in the code. We can only push and pop things onto our stack. This represents the single thread provided.  
+- The next box we see is the JS box. This is where our code runs. The call stack is where we are in the code. We can only push and pop things onto our stack. This represents the 🧵 `single-thread` 🧵 provided.
+![Event Loop Javascript](assets/eventloopjavascript.png)
 
-Our WebAPIs, is where the _magic_ happens. This is where our DOM lives, and our asynchronous calls such as SetTimeout, SetInterval, AJAX calls, and our event listeners. The WebAPIs are effectively threads in our JS.  
+- Our WebAPIs, is where the 🔮*magic*🔮 happens. This is where our DOM lives, and our asynchronous calls such as SetTimeout, SetInterval, AJAX calls, and our event listeners. The WebAPIs are effectively threads in our JS. 
```

Comment:
Uh oh! We said that JS is single-threaded, but now there are multiple threads? 

How can we simply either explain this or avoid going into this much depth?

My inclination is to explain as little as possible and focus on what helps students be better coders in the here and now. But also exposing students to concepts is also important.

The event loop is so tricky to write about. I think you're doing a fantastic job!

Reply:
The single-threaded vs asynchronous concept is introduced and fleshed out a bit earlier, but I agree that there's already sooo much in this module that it would be better to keep things as simple as possible. I never mastered that understanding and I finished a bootcamp and have a job, lol. The single-threaded/asynchronous javascript point was in the original README I am working off of and I kept it at the beginning of the `# Objective 3: The Event Loop` section. Would you all (@Krafalski and @mikeboyle ) take one more look at that chunk and see if it's explained simply enough, or if you'd like me to simplify more, or even cut it out

Reply:
I think I am looking at the updated version - but just to be sure I would strike this entire sentence:

```
The WebAPIs are effectively threads in our JS.
```

I really like:
```
moved over to the WebAPI'sWebAPIs area until it resolves.

```
I think it is the right amount of depth.

You can always provide a link to [What the heck is the event loop anyway](https://www.youtube.com/watch?v=8aGhZQkoFbQ) / other references for students who want to go into more depth. For current students, a bit of handwaving `this is the level of depth we want you to try to understand today, this takes a while to learn/understand. It's ok if it doesn't all click today`

I would definitely lean on your own experiences both as a student and as someone who works in the field to help determine the right amount of depth for the material. I think it is very easy to add more details/information and it is easy for curriculum to get over-stuffed. Any place you feel there could be simplifying/streamlining is worth considering. 

Students can only learn so much in a day and to me, is more important we are focusing on the most important things, rather than all the things.